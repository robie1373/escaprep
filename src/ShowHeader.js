import React, { Component } from 'react';
import logo from './logo.svg';

class ShowHeader extends Component {
  render () {
    return (
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <h1 className="App-title">Escalation Preparation</h1>
      </header>
    );
  }
}

export default ShowHeader;
