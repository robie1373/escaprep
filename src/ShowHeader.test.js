import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ShowHeader from './ShowHeader';

test('snapshot', () => {
  const component = renderer.create(<ShowHeader />);

  let tree = component.toJSON()
  ;
  expect(tree).toMatchSnapshot();
});
