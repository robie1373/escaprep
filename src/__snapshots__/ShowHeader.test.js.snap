// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h1
    className="App-title"
  >
    Escalation Preparation
  </h1>
</header>
`;
